docs/remote-authentication/remote-authentication.md

```diff
@@ -15,12 +15,13 @@ SystemWebAdapterConfiguration.AddSystemWebAdapters(this)
     .AddProxySupport(options => options.UseForwardedHeaders = true)
     .AddRemoteApp(options =>
     {
-        options.ApiKey = "MySecretKey";
+        // ApiKey is a string representing a GUID
+        options.ApiKey = "00000000-0000-0000-0000-000000000000";
```

Comment:
Maybe just use `Guid.NewGuid()` so that it works, but they'll need to change it for their own to actually connect. This seems like something people would copy, but not change

Reply:
Or, even better, why not show it being access from configuration?

Reply:
The validation will reject an empty guid, so this will fail if they copy it.